cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -89,6 +89,30 @@ var (
 		"name",  // name of the repository that was indexed
 	})
 
+	metricIndexingDelay = promauto.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
If this produces more interpretable results on dot com, I'll open another PR to remove the old metric and adjust the buckets here.